.github/workflows/ci.yaml

```diff
@@ -95,6 +95,10 @@ jobs:
       uses: py-actions/py-dependency-install@v4
       with:
         path: requirements/ci.txt
+    - name: Authorize ssh agent
+      uses: webfactory/ssh-agent@v0.7.0
+      with:
+        ssh-private-key: ${{ secrets.MLOPS_TEST_ACTION_SSH_KEY }}
```

Comment:
do we need to keep this?

Reply:
Actually, yes, since Neuro-Flow clones repo via ssh. So even for public repos ssh keys are required.
I could replace it with (1) turning repo to public and (2) auto-generating ssh keys. wdyt?

Reply:
if you think this is what we need - ok. I just thought this might have been some debug leftover.

Reply:
I think another test case where we verify private repo actions will not hurt :)